UnitTests/Plugins/GitUIPluginInterfacesTests/PluginsPathScannerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using GitUIPluginInterfaces;
+using NUnit.Framework;
+
+namespace GitUIPluginInterfacesTests
+{
+    [TestFixture]
+    public class PluginsPathScannerTests
+    {
+        [TestCase(@"..\..\PathScanningData", "PluginInRootDir.dll", "PluginInOwnDir.dll")]
```

Comment:
Could you clarify it for me (I'm a bit slow, sorry).

Plugins are stored in `c:\Users\<user>\Local\GitExtensions\GitExtensions\UserPlugins` folder:
```
UserPlugins
  \Plugin1
      \Plugin1.dll
      \Plugin1.config
  \Plugin2
      \Plugin2.dll
      \Plugin2.config
```

In this scenario this looks like:
```
PathScanningData
  \PluginInRootDir
      \PluginInRootDir.dll
  \NotFoundPlugin
      \SubDir
          \NotFoundPlugin.dll1.dll
  \PluginInRootDir.dll
```

Why are we expecting to load `PluginInRootDir.dll`?

Reply:
Backward compatibility. I'm using the method for both core and user plugins. Is it ok? Also PM now extracts plugins in a single directory.

Reply:
Ah, makes sense, thank you.
all good